cumulusci/robotframework/utils.py

```diff
@@ -1,7 +1,56 @@
+import time
+import functools
+from selenium.common.exceptions import ElementNotInteractableException
+from selenium.common.exceptions import StaleElementReferenceException
+from selenium.common.exceptions import WebDriverException
+from SeleniumLibrary.errors import ElementNotFound
+
+
 def set_pdb_trace():
     import sys
     import pdb
 
     for attr in ("stdin", "stdout", "stderr"):
         setattr(sys, attr, getattr(sys, "__%s__" % attr))
     pdb.set_trace()
+
+
+ALWAYS_RETRY_EXCEPTIONS = (
+    ElementNotFound,
+    ElementNotInteractableException,
+    StaleElementReferenceException,
+)
+
+
+def selenium_retry(func):
+    @functools.wraps(func)
+    def deco_selenium_retry(self, *args, **kwargs):
+        error = None
+        try:
+            return func(self, *args, **kwargs)
+        except Exception as e:
+            error = e
+            retry = False
+            if isinstance(e, ALWAYS_RETRY_EXCEPTIONS) or (
+                isinstance(e, WebDriverException)
+                and "Other element would receive the click" in str(e)
+            ):
+                retry = True
+            if retry:
+                self.builtin.log(
+                    "Retrying call to method {}".format(func.__name__), level="WARN"
```

Comment:
is this legitimately a WARN?

Reply:
That's definitely debatable.  I put it as a warn to highlight that it might warrant investigation as to why the retry was needed.